### ISSUE TITLE
toplayer及之后的Conv2d和BatchNorm2d没有设置初始化方式？

### ISSUE THREAD

**Issue by Whu-wxy** [WenmuZhou/PSENet.pytorch#62]
看到whai362的代码中，以下代码是对于模型整体进行的初始化。
我看到这里只是在resnet参数初始化，resnet后面从toplayer到最后没有进行这样的初始化会不会导致结果较差？

if isinstance(m, nn.Conv2d):
                n = m.kernel_size[0] * m.kernel_size[1] * m.out_channels
                m.weight.data.normal_(0, math.sqrt(2. / n))
            elif isinstance(m, nn.BatchNorm2d):
                m.weight.data.fill_(1)
                m.bias.data.zero_()

**Comment by WenmuZhou** [comment]
自动初始化的
